README.md

```diff
@@ -16,9 +16,15 @@ There are also some CSS frameworks like Bootstrap [that is broadly used in ~30%
 
 ## 🌱  How to start this project
 
-1. This project comes with the necessary files to start working.
-2. Clone this repository locally or open it with gitpod for a better experience.
-3. Run the following command to preview your changes: `npx --yes http-server`
+This project comes with the necessary files to start working, but you have two options to start:
+
+a) Use Gitpod: copy & paste this on your browser: gitpod.io#<https://github.com/breatheco-de/exercise-product-landing-page-plain-css.git>
```

Comment:
a) Use gitpod: open this link in your browser to clone it with gitpod: [https://gitpod.io#https://github.com/breatheco-de/exercise-product-landing-page-plain-css](https://gitpod.io#https://github.com/breatheco-de/exercise-product-landing-page-plain-css)